app/views/books/index.html.erb

```diff
@@ -1,11 +1,24 @@
 <%= render "books/filter", filter: filter %>
 
 <% if books.any? %>
-  <ul class="books">
-    <% books.each do |book| %>
-      <li><%= link_to book.title, book %></li>
-    <% end %>
-  </ul>
+
+  <table class="books">
+    <thead>
+      <tr>
+        <th><%= link_to("Title", sort_path("title")) %></th>
```

Comment:
The helper is needed to retain the current page number and filter when changing the sort order.